apps/filebrowser/src/filebrowser/utils.py

```diff
@@ -16,11 +16,20 @@
 import io
 import os
 import logging
-LOG = logging.getLogger()
+from datetime import datetime
 
+from desktop.conf import TASK_SERVER_V2
+from desktop.lib.django_util import JsonResponse
 from filebrowser.conf import ARCHIVE_UPLOAD_TEMPDIR
+
+if hasattr(TASK_SERVER_V2, 'get') and TASK_SERVER_V2.ENABLED.get():
+  from desktop.settings import parse_broker_url
+LOG = logging.getLogger()
```

Comment:
Nit: space above

Reply:
Added a space based on ruff's suggestion. 
`apps/filebrowser/src/filebrowser/utils.py:16:1: I001 [*] Import block is un-sorted or un-formatted
Found 1 error.
`